dts/arm/ti/mspm0g1x0x_g3x0x/mspm0g1x0x_g3x0x-pinctrl.dtsi

```diff
@@ -0,0 +1,146 @@
+#include <dt-bindings/pinctrl/mspm0-pinctrl.h>
+
+&pinctrl {
+
+	/omit-if-no-ref/ i2c0_scl_pa1_pull_up: i2c0_scl_pa1_pull_up {
+		pinmux = <MSP_PINMUX(2,MSPM0_PIN_FUNCTION_3)>;
+		input-enable;
+		drive-open-drain;
+	};
+
+	/omit-if-no-ref/ i2c0_sda_pa0_pull_up: i2c0_sda_pa0_pull_up {
+		pinmux = <MSP_PINMUX(1,MSPM0_PIN_FUNCTION_3)>;
+		input-enable;
+		drive-open-drain;
+	};
+
+	/omit-if-no-ref/ i2c1_sda_pa10: i2c1_sda_pa10 {
```

Comment:
Am not sure why `_pull_down/up` is missing here. Is it because the pin is `High-Drive`? Otherwise we don't need to keep a separate suffix for pull_up for all the I2C.